versaepoch/src/components/ui/TableSortPanel.tsx

```
@@ -0,0 +1,98 @@
+  const columns = table.getAllColumns().filter((col) => col.getCanSort());
+
+  const availableColumns = columns
+    .filter((col) => !sortingState.find((sort) => sort.id === col.id))
+    .map((col) => ({
+      id: col.id,
+      label: capitalizeWord(col.id),
+    }));
+
+  const handleAddSort = (columnId: string) => {
+    const column = table.getColumn(columnId);
+    if (column && column.getCanSort()) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)